VoiceController.php

```diff
@@ -1,48 +1,94 @@
-public function voice(Request $request){
+<?php
+
+function voice(Request $request)
+{
     $request->validate([
         'question_id'=>'required|int|exists:questions,id',
         'value'=>'required|boolean',
     ]);
+    
+    $AuthUser = auth()->user();
```

Comment:
@muhamedRadwan not sure if this variable is very useful, you can always use `auth()->user()->xxxxx()` everywhere

Reply:
I thought it every time i use auth()->user() it will get the user from its db table not caching them

Reply:
I don't think so.